packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -276,7 +286,8 @@ export function RecordTable_Sequences(
       leaves == null ||
       tree == null ||
       filteredRows == null ||
-      filteredRows.length === 0
+      filteredRows.length === 0 ||
+      filteredRows.length > MAX_SEQUENCES_FOR_TREE
```

Comment:
We were filtering the tree even when we're not showing it. Duh! Not any more.

Probably need to add `< MIN_SEQUENCES_FOR_TREE` also.